beacon-chain/core/helpers/committee.go

```diff
@@ -383,10 +385,200 @@ func UpdateProposerIndicesInCache(state state.ReadOnlyBeaconState) error {
 	})
 }
 
-// ClearCache clears the committee cache
+// ClearCache clears the beacon committee cache and sync committee cache.
 func ClearCache() {
 	committeeCache = cache.NewCommitteesCache()
 	proposerIndicesCache = cache.NewProposerIndicesCache()
+	syncCommitteeCache = cache.NewSyncCommittee()
+}
+
+// IsCurrentPeriodSyncCommittee returns true if the input validator index belongs in the current period sync committee
+// along with the sync committee root.
+// 1.) Checks if the public key exists in the sync committee cache
+// 2.) If 1 fails, checks if the public key exists in the input current sync committee object
```

Comment:
> along with the sync committee root

What does this mean? Was this supposed to return the root?

Reply:
Bad sentence. It means `using the sync period root as key`. Will fix